dashboard/migrations/0017_auto_20200720_1203.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.14 on 2020-07-20 16:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0015_auto_20200116_1408'),
```

Comment:
Needs to go after the utf8 conversion now.

```suggestion
        ('dashboard', '0016_utf8_conversion'),
```